apps/generator-cli/src/app/mocks/passthrough-command.mock.ts

```diff
@@ -0,0 +1,17 @@
+export class PassthroughCommandMock {
+  name = jest.fn().mockReturnValue(this._name);
+  options: { [key: string]: string } = {};
```

Comment:
Could you use `Record<string, string>` instead, please?

Reply:
done